EXAMPLE/group_vars/_skel/cluster_vars.yml

```diff
@@ -7,7 +7,13 @@ gcp_credentials_json: "{{ lookup('file', gcp_credentials_file) | default({'proje
 app_name: "test"                  # The name of the application cluster (e.g. 'couchbase', 'nginx'); becomes part of cluster_name.
 app_class: "test"                 # The class of application (e.g. 'database', 'webserver'); becomes part of the fqdn
 
-beats_target_hosts: []            # The destination hosts for e.g. filebeat/ metricbeat logs
+beats_config:
+  filebeat:
+#    output_logstash_hosts: ["localhost:5044"]   # The destination hosts for filebeat-gathered logs
```

Comment:
Duplicating `output_logstash_hosts` - Will we have two separate logstash hosts for FB & MB simultaneously?  Or could we perhaps have one mention of `output_logstash_hosts` which both FB & MB would inherit if the endpoint is the same. i.e.

```
beats_config:
  output_logstash_hosts: 
  filebeat:
    extra_logs_paths:
  metricbeat:
    extra_logs_paths:
```

Reply:
Discussed on teams - existing config helps with having two different endpoints for another use case. LGTM!